test/commontransformations.jl

```diff
@@ -333,11 +345,11 @@
             trans = euler_rotation(0.1,0.2,0.3)
             x2 = Point(2.730537054338937,0.8047190852558106,2.428290466296628)
 
-            @test trans.t1.t1 == RotationXY(0.1)
-            @test trans.t1.t2 == RotationYZ(0.2)
-            @test trans.t2 == RotationZX(0.3)
+            #@test trans.t1.t1 == RotationXY(0.1)
+            #@test trans.t1.t2 == RotationYZ(0.2)
+            #@test trans.t2 == RotationZX(0.3)
```

Comment:
I'll clean all this up when rotations are changed. For now it is a reminder of the old behaviour while I consider how we might want axis rotations to compose into Euler angles (or just into a matrix).